dmutils/email.py

```diff
@@ -27,25 +25,21 @@ def send_email(
             'from_name': from_name,
             'to': [{
                 'email': email_address,
-                'name': 'Recipient Name',
                 'type': 'to'
-            }],
+            } for email_address in to_email_addresses],
             'important': False,
-            'track_opens': None,
-            'track_clicks': None,
+            'track_opens': False,
+            'track_clicks': False,
             'auto_text': True,
             'tags': tags,
-            'headers': {'Reply-To': from_email},  # noqa
+            'headers': {'Reply-To': from_email},
+            'preserve_recipients': False,
             'recipient_metadata': [{
                 'rcpt': email_address
-            }]
+            } for email_address in to_email_addresses]
         }
 
-        result = mandrill_client.messages.send(
-            message=message,
-            async=False,
-            ip_pool='Main Pool'
-        )
+        result = mandrill_client.messages.send(message=message, async=True)
```

Comment:
We should set up a webhook so that we're at least notified of failures. I don't think it's a blocker for this PR.